data/changelog.txt

```diff
@@ -1,3 +1,568 @@
+Aftershock has new fauna monsters
+More atmospheric lore and details for Aftershock
+Dinomod has expanded it's dialogue options with mainline NPCs
+The DinoLab
+The Forge of Wonders now has three shops and stronger defenses on the vault
+The No Hope mod is available to greatly reduce the average player lifespan
+Dinomod specific achievements as well as Aftershock specific achievements
+Aftershock: Translocators for semi-convenient long range teleporting.
+Burned Zombie Dinos
+New gizmos and gadgets in Aftershock
+Aftershock: Stratoscomm Relay Station
+[Magiclysm] Adds new zombie animals
+Add mythos monsters to Mythos mod
+Dinomod Apex predators don't appear randomly day one
+Adds TropiCataclysm to the repository
+craftable holiday sweaters for dinos
+Hylian Adventurer Profession for Mythical Martial Arts
+Many south american animals added to Tropicataclysm
+Adds an innawood mod for wilderness survival.
+Aftershock Exoplanet: Glacial tunnels, a new low-risk area and the fauna that inhabits them
+TropiCataclysm Adds Termites and new content for Triffids
+Adds cast-bronze tools to Innawoods mod.
+[Innawood] Chemistry, electricity, and more!
+Many new dinosaurs added to Dinomod
+A mod to add subtitles to Rubik's dialogue in English.
+Tameable Wildlife MOD
+Dark Days of the Dead: Headshot requirements and deadly zombie bites are in!
+Dinomod NPC Missions
+New Mod Xedra Evolved with new magic classes, monsters, professions and CBMs plus Cyborg professions
+Desert region revival
+Add Megafauna Mod
+[DinoMod] tyrannosaurus bio-operator
+[DinoMod] Nanuqsaurus and burned rebuild
+Aftershock: Add functional jet packs
+Xedra Evolved: New NPCs to meet and rescue
+[DinoMod] player mutations change monster faction behavior
+[DinoMod] stegoceras
+Magiclysm Adding new magic plants
+[DinoMod] compsognathus, eoraptor, amargasaurus bio-operators
+Magiclysm Quest & Lore patch to item distribution
+[DinoMod] mi-go scout tower dino variant
+The Backrooms mod
+Vampire Virus in Xedra Evolved
+Crazy Cataclysm: Adds Powerthirst and Rocket Cans
+Xedra Evolved magic classes Eaters, Dreamers, Inventors and Dreamsmiths
+[Magiclysm] Make Summoning Great Again!
+Implementing more proper alchemy and some another stuff
+Aftershock Space items and smartguns
+Aftershock New CBM and class
+Aftershock: New Monofilament Whip CBM
+Butchering candy monsters gives candy flesh
+Add standard combat test mod
+Nested Crafting Categories
+Add cosmetic variants for mutations
+
+## Balance:
+Mutation System Overhaul
+Made windmills able to grind up dried rice, cooked acorn meal, and roasted cattail rhizome into flour
+Shocker zombies cannot be harvested for bionics in vanilla
+Reduce crafting proficiency penalties based on partial proficiency level
+Updated brewing recipes and fermentation times to be more realistic
+add stamina cost to archery
+Change Cut Cotton to Patchwork, add Patchwork to Rags
+monster HP scaling section in GAME_BALANCE
```

Comment:
```suggestion
```

Someone mislabled their PR. This should be in Documentation.

Reply:
Is there a documentation section for the changelog? If not I'll just remove this one.

Reply:
I dont think so and there should be none because this is, although it is tedious and much appreciated work, relevant to the player.

Reply:
OK I've added a new documentation section right before Mods section.  Please let me know if there are any other major documentation PRs that need to be included.